### DISCUSSION TITLE
Image optimisation wrongly displayed and calculated

### DISCUSSION THREAD

**Discussion by marcelscruz** [vercel/community#1333] [Help] [answered]
### Summary

It seems that the calculation and display of image optimisation are not accurate on the dashboard.

Firstly, renamed projects are showing as duplicated, and so are their counts.
In the examples below, `find-a-job-dev` was renamed to `jobstache` and `public-apis-site` to `public-apis`:
<img width="312" alt="Screenshot 2023-01-23 at 13 42 04" src="https://user-images.githubusercontent.com/12413903/214042625-95909470-b8cf-49ee-8c5e-d31db812b678.png">
<img width="323" alt="Screenshot 2023-01-23 at 13 42 13" src="https://user-images.githubusercontent.com/12413903/214042632-c0e90dcc-4b51-4aab-97ca-38cf37fa46b0.png">

Secondly, the `Counts` and `Projects` tab show different results:
![Screen_Recording_2023-01-23_at_13_39_14_AdobeExpress](https://user-images.githubusercontent.com/12413903/214045740-8f0850c7-7412-41a8-a66f-5c8e448480ef.gif)

Lastly, and maybe not as impactful, hovering on different elements in sequence seems to break the list of projects displayed on the popup, as they start being added together:
![Screen_Recording_2023-01-23_at_13_39_58_AdobeExpress](https://user-images.githubusercontent.com/12413903/214045702-9a50bb9c-7c83-4f96-8c8e-b7ff17ac39b5.gif)

I'm happy to provide more examples or information if needed.

Thanks!

**Comment by calebboyd** [comment, marked as answer]
Hi @marcelscruz, Thank you for the report! I think these issues may all be related to the first one here, where renamed projects are duplicated - Which should be fixed! If you're still seeing this Please let us know.

**Comment by marcelscruz** [comment]
Hi @calebboyd, thanks for letting me know.

Yes, it seems to be fixed indeed.

Thank you for the quick resolution. :)
